tools/aiecc/aiecc/main.py

```diff
@@ -144,6 +144,10 @@ def process_arm_cgen():
       if(opts.xaie == 2):
           do_call(['aie-translate', '--aie-generate-xaie', '--xaie-target=v2', file_physical, '-o', file_inc_cpp])
       else:
+        file_inc_cpp = os.path.join(tmpdirname, 'aie_inc.cpp')
```

Comment:
What are the changes here L147-150 for?

Reply:
I think that asking for an airbin is mutually exclusive with asking for an aie_inc.cpp, but that may not be the best option.

Reply:
I think I am missing the path to generate the airbin here